sentinel-2/kndvi/README.md

```diff
@@ -0,0 +1,52 @@
+---
+layout: kNDVI with Cloud Masking
+parent: Sentinel-2
+title: Contribute
+grand_parent: Sentinel
+layout: script
+permalink: /sentinel-2/kndvi/
+nav_exclude: true
+nav_order: 99
+has_children: false
+permalink: /contribute/
+---
+
+# Evaluate and Visualize
```

Comment:
You could do this following the [example](https://github.com/sentinel-hub/custom-scripts/blob/main/contribute/example-multiple-scripts/index.md#adding-examples) in the contribution markdown.

Reply:
using the evalscripturl parameter requires the evalscript to be online already? Or should I do it by Base64 encoding? I have no experience with that

Reply:
Yes, it requires the evalscript online, which will be the one you're adding with this PR. No worries if it does not work when testing locally.

Reply:
so for the script url, I can just take https://github.com/sentinel-hub/custom-scripts/blob/main/sentinel-2/kdvi/script.js ,right?

Reply:
It should be `https://custom-scripts.sentinel-hub.com/custom-scripts/sentinel-2/kndvi/script.js` as shown in the ndvi example I provided below.